packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/http-request-action/components/KeyValuePairInput.tsx

```diff
@@ -31,6 +25,13 @@ const StyledPlaceholder = styled.div`
   height: ${({ theme }) => theme.spacing(8)};
 `;
 
+const StyledFormTextFieldInputContainer = styled.div`
+  width: 100%;
+  .ProseMirror {
```

Comment:
This isn't how we usually do it in the codebase, did you try to find a solution that avoids harcoding the classname? Sometimes there is no way around but in many case there is.